README.md

```diff
@@ -0,0 +1,105 @@
+# Glogger
+
+Glogger is the logger for mia-platform go services.
+
+It uses [logrus](https://github.com/sirupsen/logrus) as logging library,
+and implements a middleware to be used with [http gorilla mux router](https://github.com/gorilla/mux).
+
+This library follow the Mia Platform logging guidelines.
+
+## Install
+
+This library require golang at version >= 1.13
+
+```sh
+go get -u github.com/mia-platform/glogger
+```
+
+## Example usage
+
+### Basic logger initialization.
+
+The allowed log level are those parsed by [logrus ParseLevel](https://godoc.org/github.com/sirupsen/logrus#ParseLevel) (e.g. panic, fatal, error, warn, warning, info, debug, trace).
+
+```go
+// Logger setup
+log, err := glogger.InitHelper(logger.InitOptions{Level: "info"})
+if err != nil {
+  log.Fatal(err.Error())
+}
+```
+
+### Setup log middleware
+
+Init log middleware for [mux router](https://github.com/gorilla/mux). This log the `incoming request` and `request completed` following the mia-platform guidelines.
+
+```go
+r := mux.NewRouter()
+r.Use(glogger.RequestMiddlewareLogger(log, nil))
+```
+
+and, to retrieve logger injected in request context:
+
+```go
+func(w http.ResponseWriter, req *http.Request) {
+  loggerFn := logger.Get(req.Context())
+  loggerFn.Info("log message")
+}
+```
+
+#### with excluded path
+
+You can restrict the path where the logger middleware take effect using the second paramenter in RequestMiddlewareLogger. For example, this could be useful to exclude `incoming request` and `request completed` logging in path router.
```

Comment:
```suggestion
You can restrict the path where the logger middleware take effect using the second paramenter in `RequestMiddlewareLogger`. For example, this could be useful to exclude `incoming request` and `request completed` logging in path router.
```